<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PsdUI</class>
 <widget class="QMainWindow" name="PsdUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PoseDriver_v2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="parentObj_btn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #116776, stop:0 #116776);
padding: 5px;
border-radius: 5px;
}

QPushButton:hover
{
background-color: #1d8ba3;
border-radius: 5px;
}

QPushButton:pressed
{
background-color: #1d1d1d;
border-radius: 5px;
}

QPushButton:disabled
{
background-color: #4b4b4b;
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Parent Obj &gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="parentObj_line"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="followObj_btn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #116776, stop:0 #116776);
padding: 5px;
border-radius: 5px;
}

QPushButton:hover
{
background-color: #1d8ba3;
border-radius: 5px;
}

QPushButton:pressed
{
background-color: #1d1d1d;
border-radius: 5px;
}

QPushButton:disabled
{
background-color: #4b4b4b;
border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Follow Obj &gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="followObj_line"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="lside_rb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rside_rb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mside_rb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>M</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="axis_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="axis_cb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>+x</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+y</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+z</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-x</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-y</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-z</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="scale_lb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Scale:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="scalePSD_dsb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhFormattedNumbersOnly</set>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="value">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createPsd_btn">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:1 #116776, stop:0 #116776);
padding: 5px;
border-radius: 5px;
}

QPushButton:hover
{
background-color: #1d8ba3;
border-radius: 5px;
}

QPushButton:pressed
{
background-color: #1d1d1d;
border-radius: 5px;
}

QPushButton:disabled
{
background-color: #4b4b4b;
border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
